api/libproject/devstart.rb

```diff
@@ -888,12 +910,23 @@ def migrate_workbench_data()
   end
 end
 
-def load_config(project)
+def get_config(project)
   configs = {
     TEST_PROJECT => "config_test.json",
-    "aou-res-workbench-stable" => "config_stable.json",
+    "all-of-us-rw-stable" => "config_stable.json",
   }
-  config_json = configs[project]
+  return configs[project]
+end
+
+def get_auth_domain(project)
+  config_json = get_config(project)
+  config_line = `grep registeredDomainName config/config_stable.json`
```

Comment:
Pretty dirty, how about:

```
JSON.parse(File.read("config/#{config_json}"))["firecloud"]["registeredDomainName"]
```

(also, need to parameterize on config_json here, currently hardcoded to stable)

Reply:
Good call. Done.